index.js

```diff
@@ -38,12 +38,13 @@ app.use(function(req, res, next) {
     }
 });
 
-app.get('/:date', function(req, res, next) {
-    var date = moment(req.params.date, 'YYYY-MM-DD').tz('Europe/Helsinki');
+app.get('/:date/:lang', function(req, res, next) {
```

Comment:
For backwards compatibility the route should also respond to `/:date` and default the language to Finnish for example. Take look at https://github.com/tophattom/ruoka-api/blob/feature/other-campuses/index.js#L41 for how I did the campus argument defaulting to TUT.